validate.sh

```diff
@@ -124,7 +131,7 @@ timed() {
 print_header() {
     TITLE=$1
     TITLEPAT="$(echo "$TITLE"|sed 's:.:=:g')"
-    echo "$CYAN===X============================================================ $(date +%T) ===$RESET" \
+    cyan "===X========================================================================== $(date +%T) ===" \
```

Comment:
`Step Build: full build plan (cached and to-be-built dependencies):` didn't fit the previous length.